### PR TITLE
Add insert_after_yank config option

### PR DESCRIPTION
**Pull request by Yaraslaut**
Refs https://github.com/contour-terminal/contour/issues/1604

### CODE REVIEW

File: src/vtbackend/Settings.h [hunk 1]
```diff
@@ -89,6 +89,7 @@ struct Settings
     PrimaryScreen primaryScreen;
 
     bool fromSearchIntoInsertMode = true;
+    bool inInsertAfterYank = true;
```
Comment (christianparpart) [inline src/vtbackend/Settings.h:92]: Here it defaults to `true`, but above (in `src/contour/Config.h`), the value is defaulted to `false`. I think it should both be the same (my preference would be `false`, to retain current semantics and standard vi-experience).

Comment (Yaraslaut) [reply src/vtbackend/Settings.h:92]: You are right, I was using it for testing purposes, I still need to connect config and settings

**Comment by christianparpart** [comment]
Should we mention this in the documentation (website) as well?

**Comment by Yaraslaut** [comment]
> Should we mention this in the documentation (website) as well?

Oh, right
We should optimize it away xD

**Review by christianparpart — APPROVED** [review]
Thank you

**Pull request merged**